domains/spicyaxolotl.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "My personal website",
+  "repo": "https://github.com/Spicy-Axolotl/spicy-axolotl.github.io",
+  "owner": {
+    "username": "Spicy-Axolotl",
+    "email": "",
```

Comment:
hold on, can i get some contact details such as your discord/your email/etc beside of your github pls?

Reply:
Here is my discord: Axolotl (mycoolface256)#4736

Reply:
no, write them into the json file.

Reply:
There you go.

Reply:
also feel free to join our discord if you are interested

Reply:
Ok.